modules/logging-loki-reliability-hardening.adoc

```diff
@@ -0,0 +1,39 @@
+// Module included in the following assemblies:
+//
+// * logging/cluster-logging-loki.adoc
+
+:_content-type: REFERENCE
+[id="logging-loki-reliability-hardening_{context}"]
+= Configuring Loki to tolerate node failure
+
+In logging 5.8 and newer versions, the Loki Operator supports setting pod anti-affinity rules to request that pods of the same component are scheduled on different available nodes in the cluster.
+
+include::snippets/about-pod-affinity.adoc[]
+
+The Operator sets default, preferred `podAntiAffinity` rules for all Loki components, which includes the `compactor`, `distributor`, `gateway`, `indexGateway`, `ingester`, `querier`, `queryFrontend`, and `ruler` components.
```

Comment:
Possible candidate for an unordered list, but I also don't know if I actually like that better or not (taking up extra real estate).

Reply:
I think I prefer this one as is 🙂